google/cloud/bigquery/v2/minimal/internal/job_request.cc

```diff
@@ -163,12 +169,14 @@ StatusOr<rest_internal::RestRequest> BuildRestRequest(
     request.AddQueryParameter("maxResults", std::to_string(r.max_results()));
   }
   if (r.min_creation_time()) {
-    request.AddQueryParameter("minCreationTime",
-                              internal::FormatRfc3339(*r.min_creation_time()));
+    request.AddQueryParameter(
+        "minCreationTime",
+        std::to_string(CastTimeToMilliseconds(r.min_creation_time().value())));
```

Comment:
```suggestion
        std::to_string(CastTimeToMilliseconds(*r.min_creation_time())));
```

You already validated that `operator*` is safe.